sse2neon.h

```diff
@@ -9122,43 +9122,151 @@ FORCE_INLINE __m128i _mm_aesenclast_si128(__m128i a, __m128i RoundKey)
 #endif
 }
 
+FORCE_INLINE uint8x16_t _sse2neon_vqtbl1q_u8(uint8x16_t t, uint8x16_t idx)
+{
+#if defined(__aarch64__)
+    return vqtbl1q_u8(t, idx);
+#else
+    // Split `idx` into two D registers.
```

Comment:
Use `'idx'` instead of the form with [backtick](https://en.wikipedia.org/wiki/Backtick) since some terminals can not render it properly.